config-store-yaml/pom.xml

```diff
@@ -49,6 +43,11 @@
         <version>3.17.2</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>com.datastax.oss</groupId>
+        <artifactId>java-driver-shaded-guava</artifactId>
+	<scope>provided</scope>
```

Comment:
bad formatting?

Reply:
Accidental tab it seems, will fix.

Reply:
And on scope/import: as per our out-of-band discussion, only test dependency here for which no Imports needed.